keyboards/snes_macropad/info.json

```diff
@@ -0,0 +1,79 @@
+{
+    "manufacturer": "JBarberU's",
+    "keyboard_name": "SNES Macropad",
+    "maintainer": "jbarberu",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "oled": true
+    },
+    "ws2812": {
+        "pin": "GP5",
+        "driver": "vendor"
+    },
+    "matrix_pins": {
```

Comment:
just a heads up, you could specify the number of columns and rows instead of adding the `NO_PIN`s here.